src/Packetery/Module/Checkout.php

```diff
@@ -29,6 +29,7 @@ class Checkout {
 	const ATTR_POINT_CITY   = 'packetery_point_city';
 	const ATTR_POINT_ZIP    = 'packetery_point_zip';
 	const ATTR_POINT_STREET = 'packetery_point_street';
+	const ATTR_POINT_PLACE  = 'packetery_point_place';
```

Comment:
Zvážil bych pojmenovat to nějak pochopitelněji (třeba ATTR_BUSINESS_NAME), nebo opatřit komentářem.

Reply:
widget tomu říká place

Reply:
Vím. Konzultoval jsem s Jardou. Přijatelné je např. opatřit konstantu komentářem.